include/tsutil/StringConvert.h

```diff
@@ -0,0 +1,71 @@
+/*
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+#pragma once
+
+#include <charconv>
+#include <stdexcept>
+#include <string>
+#include <system_error>
+
+namespace ts
+{
+
+inline std::string
+hex(const std::string_view input)
```

Comment:
I was initially confused, assuming `hex` converted from base 10 or some such to hex. Maybe comment with something like:

```cpp
/** Convert values into their hex ascii equivalents.
 * @note This is a reimplementation of boost's hex.
 * @return The series of hex values corresponding to @a input
 */
```